CONTRIBUTING.md

```diff
@@ -1,8 +1,6 @@
 # Contributing to PatternFly React
```

Comment:
looks like the contributor covenant link at the end of this file leads to a 404 and could be updated to https://www.contributor-covenant.org/version/1/4/code-of-conduct/code_of_conduct.md (if we want to still use v1.4; they're currently on 2.1)